website/src/app/docs/page.tsx

```
@@ -774,10 +773,11 @@ <h1 class="hero-title">
         </div>
     </div>
 
-    <!-- Go back button uses browser history -->
+    {/* Go back button uses browser history */}
     <script>
         // you can customize behavior here if needed
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)